src/Workspaces/MSBuild/Core/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -22,25 +22,13 @@
     </PackageDescription>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" />
+    <PackageReference Include="Microsoft.Build.Framework" />
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Newtonsoft.Json" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
     <ProjectReference Include="..\..\Core\Portable\Microsoft.CodeAnalysis.Workspaces.csproj" />
-
-    <!-- For the following package we directly include the binary into this NuGet package because we don't want/need a separate NuGet package
-         for it. PrivateAssets="all" is needed to prevent this reference from becoming a package reference in the package, as a workaround for
-         https://github.com/NuGet/Home/issues/3891.
-         -->
-    <ProjectReference Include="..\BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj" PrivateAssets="all" ReferenceOutputAssembly="false">
```

Comment:
I wonder if the lack of a ProjectReference here might have problems in VS understanding there's still an implicit dependency, but maybe it's fine...